tests/test_bitinformation_pipeline.py

```diff
@@ -27,6 +27,18 @@ def test_full():
     assert bitrounded_compressed_size < compressed_size
 
 
+def test_full_max_keepbits():
+    """Test pipeline to get maximum keepbits"""
+    label = "air_temperature"
+    ds = xr.tutorial.load_dataset(label)
+    bi = xb.get_bitinformation(ds)
+    kb = xb.get_keepbits(bi)
+    kb_max = kb.max(dim="dim")
+    _ = xb.plot_bitinformation(bi.isel(dim=slice(0, 1)))
```

Comment:
why slice(0,1)?

Reply:
To keep the `dim`-dimension. I will add also the case, when `dim` only remains as a coordinate.

Reply:
ah ok. I usually do `.isel(dim=[0])` which is shorter. I think it would be easier if dim doesnt need to be a dimension here.

Reply:
`dim` does not need to be a dimension here. I just added it to make sure to test that both cases work. (though I still need to add the other test case as well)